primitives/src/vid/advz/payload_prover.rs

```diff
@@ -67,7 +67,7 @@ pub struct LargeRangeProof<F> {
 impl<E, H> PayloadProver<SmallRangeProof<KzgProof<E>>> for Advz<E, H>
 where
     E: Pairing,
-    H: HasherDigest,
+    H: HasherDigest + Send + Sync,
```

Comment:
you already added `Send`, `Sync` to `HasherDigest` right? So these new trait bounds should be unnecessary.

Reply:
removed [607ea41](https://github.com/EspressoSystems/jellyfish/pull/496/commits/607ea411f81d2719f5d20adbe6553797c83a1a44)